cmake/externals.cmake

```diff
@@ -383,6 +383,12 @@ function(drake_add_external PROJECT)
     drake_add_foreign_external(${PROJECT})
   endif()
 
+  if(SKIP_DRAKE_BUILD)
```

Comment:
p.s. I tried *really hard* when I originally refactored all this stuff to avoid any hard-coding for specific projects... would be great if we could continue that